apps/container/src/router/index.ts

```diff
@@ -144,19 +149,34 @@ router.beforeEach(async (to, _from, next) => {
   }
 
   if (to.matched.some((record) => record.meta.auth === true)) {
-    if (authStore.getState().token) {
+    // if user doens't have a token, redirect
+    if (!authStore.getState().token) {
+      if (process.env.VUE_APP_MF_ENV !== 'local') {
+        return (window.location.pathname = '/');
+      }
       return next();
     }
-    if (process.env.VUE_APP_MF_ENV !== 'local') {
-      return (window.location.pathname = '/');
+
+    // if user needs to be confirmed and it is, don't redirect
+    if (to.matched.some((record) => record.meta.confirmed === true)) {
+      if (authStore.getState().user?.confirmed) {
+        return next();
+      }
+      return next('/review');
+    }
+
+    // if user doesn't need to be confirmed and it is, redirect
+    if (authStore.getState().user?.confirmed && !to.matched.some((record) => record.path === '/reviews') ) {
+      return next('/review');
     }
     return next();
   }
```

Comment:
não entendi esse redirect final para reviews, qual a necessidade?

Reply:
se ele tiver confirmado e não for reviews, jogue ele pra reviews, é isso que eu li